include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -225,6 +225,27 @@ class MGConstrainedDoFs : public Subscriptor
   const AffineConstraints<double> &
   get_user_constraint_matrix(const unsigned int level) const;
 
+  /**
+   * Merge selected constraints of a level into a single AffineConstraints
+   * object.
+   *
+   * @param constraints AffineConstraints object to be filled.
+   * @param level Refinement to be considered.
+   * @param add_boundary_indices Add boundary indices.
+   * @param add_refinement_edge_indices Add refinement-edge indices.
+   * @param add_level_constraints Add level constraints including the one passed
+   *   during initialize() and periodicy constraints.
+   * @param add_user_constraints Add user constraints.
+   */
+  template <typename Number>
+  void
+  merge_constraints(AffineConstraints<Number> &constraints,
+                    const unsigned int         level,
+                    const bool                 add_boundary_indices,
```

Comment:
Four bools is not a great interface in my opinion, but I understand that adding an enum or something might be a bit too much here.